lab_2/main.py

```diff
@@ -24,7 +36,15 @@ def get_language_profiles(texts_corpus: list, language_labels: list) -> dict or
     :param language_labels: a list of given language labels
     :return: a dictionary of dictionaries - language profiles
     """
-    pass
+
+    if (not isinstance(texts_corpus, list) or
+            not isinstance(language_labels, list) or not all(isinstance(i, list) for i in texts_corpus) or
+            not all(isinstance(i, str) for i in language_labels)):
```

Comment:
what does `all` do?

Reply:
together with isinstance it checks if all elements in a list(or a dict) are of a certain type